desc/equilibrium/utils.py

```diff
@@ -181,3 +183,100 @@ def parse_axis(axis, NFP=1, sym=True, surface=None):
     else:
         raise TypeError("Got unknown axis type {}".format(axis))
     return axis
+
+
+def scale_profile(profile, inner_rho):
+    """Return a new profile whose rho=1 val is value of the old profile at inner_rho.
+
+    Args
+    ----
+        profile (Profile): Profile to scale
+        inner_rho (float): rho to take as the new LCFS rho for the new profile
+    Returns
+    -------
+        new_profile (Profile): new profile whose last value is the inner_rho value of
+        the original profile
+    """
+    params_list = []
+    modes_list = []
+    if isinstance(profile, PowerSeriesProfile):
+        for coeff, mode in zip(profile.params, profile.basis.modes):
+            l = mode[0]
+            params_list.append(coeff * inner_rho**l)
+            modes_list.append(l)
+        new_profile = PowerSeriesProfile(params=params_list, modes=modes_list)
+    elif isinstance(profile, SplineProfile):
+        n_knots = profile.params.size
+        new_knots = np.linspace(0, 1, n_knots, endpoint=True)
+        new_vals = profile(np.linspace(0, inner_rho, n_knots, endpoint=True))
+        new_profile = SplineProfile(
+            values=new_vals, knots=new_knots, method=profile._method
+        )
+    else:
+        raise NotImplementedError(
```

Comment:
Instead of implementing logic specific to each profile type, we can just do `Profile.from_values`

Reply:
But instead of throwing this `NotImplementedError` we should do `Profile.from_values` in this `else` block. That way the code will still work for other profile types. And maybe throw a warning instead. 

Also what you are doing for the `SplineProfile` is equivalent to fitting `from_values`, so it's really only the `PowerSeriesProfile` that we can do analytically.

Reply:
does `SplineProfile` have a `from_values` method? I do not see one